modules/sriov-operator-metrics.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="sriov-operator-metrics_{context}"]
+= Enabling the SR-IOV network metrics exporter
+
+The SR-IOV network metrics exporter is disabled by default. To enable the metrics exporter, you must set the `spec.featureGates.metricsExporter` field to `true`.
+
+[IMPORTANT]
+====
+When the metrics exporter is enabled, the SR-IOV Network Oprator deploys the metrics exporter only on nodes that include the `feature.node.kubernetes.io/network-sriov.capable": "true"` label.
```

Comment:
Though this is probably true for most of the deployment, it's better to be more accurate: the metric-exporter is deployed on the same node as the config-daemon, which are the one described by the field `SriovOperatorConfig.Spec.ConfigDaemonNodeSelector`.

If you prefer, we can simply state something like "the metric-exporter is deployed only on node with SR-IOV capabilities"

Reply:
Updated as per the suggestion.